src/mesido/pycml/component_library/milp/electricity/electricity_cable.py

```diff
@@ -31,8 +31,13 @@ def __init__(self, name, **modifiers):
         self.nominal_current = nan
         self.nominal_voltage = nan
         self.r = 1.0e-6 * self.length  # TODO: temporary value
-        self.nominal_voltage_loss = (self.nominal_current * self.r * self.nominal_current) ** 0.5
-        self.add_variable(Variable, "Power_loss", min=0.0, nominal=self.r * self.max_current**2)
+        self.nominal_voltage_loss = (self.nominal_current * self.r * self.nominal_voltage) ** 0.5
+        self.loss_nominal = self.r * self.max_current * self.nominal_current
```

Comment:
From the variable name it is now unclear what on which variable this loss is occurring, maybe change it to power_loss_nominal. And furthermore consider whether this parameter needs to be saved separately as it is also set the nominal of the power_loss

Reply:
changed the name, as we use this variable a few lines lower to check scaling ranges I suggest we keep it saved here